doc/pages/Manual.md

```diff
@@ -652,6 +651,61 @@ other system).
 
 `FOO += ""`, `FOO := ""`, etc. are all ignored - i.e. opam never adds empty segments to an existing variable.
 
+#### Environment update portability
+<a id="env-update-rewrite"></a>
+
+Some fields define an environment update portability specification. In the opam
+file it is the [`x-env-path-rewrite:`](#opamfield-x-env-path-rewrite) in the
+form:
+
+```BNF
+<environement-rewrites> ::= { <environement-rewrite> }*
+<environement-rewrite>  ::= <ident> <bool>
+                          | <ident> <separator-formula> <path-format-formula>
+
+<separator-formula>     ::= <separator-formula> "|" <separator-formula>
+                          | ( <separator-formula> )
+                          | <separator> { { <distrib-formula> }* }
+<path-format-formula>   ::= <path-format-formula> "|" <path-format-formula>
+                          | ( <path-format-formula> )
+                          | <path-format> { { <distrib-formula> }* }
+
+<separator>             ::= (") ":" (") | (") ";" (")
+<path-format>           ::= (") host" (") | (") "host-quoted" (") | (") "target" (") | (") "target-quoted" (")
+
+<distrib-formula>       ::= <distrib-formula> <logop> <distrib-formula>
+                          | "!" <distrib-formula>
+                          | "(" <distrib-formula> ")"
+<logop>                 ::= "&" | "|"
+```
+
+The `<separator>` defines the path separator to use for the variable.
+The `<path-format>` defines the way to handle variables path formatting:
+- host: use the *host* interpretation of PATHs (i.e. convert via `cygpath` on
+  Windows).
+- host-quoted: use the *host* interpretation of entries and double-quote any
+  entries which include the separator character.
+- target: use the *target* interpretation of entries (i.e. rewrite slashes to
+  backslashes on Windows).
+- target-quoted: use the *target* interpretation of entries and double-quote
+  any entries which include the separator character.
+
+Default separator is `";" | ";" { os = "win32"}` (`;` on Windows, `:` on Cygwin
+or any other system) and default format is `target` with no slashes rewrite.
```

Comment:
The wording is a bit confusing. Reading the definition for `target` above, it feels pointless to say the default format is `target` if it’s not going to do anything.
Does that actually do nothing or is there something to say for variables? In which case it might be worth being more explicit.

Reply:
`target` not explicitly specified is a no-op indeed. I've worded it like that because in the code the default is `target` with no rewrite. Maybe not worth keep it in that way in the doc.

Reply:
Should we add a word on specific case ? `PKG_CONFIG_PATH` (`:`, `target-quoted`) and `PATH` on windows (`;`, `target_quoted`)?

Reply:
> Maybe not worth keep it in that way in the doc.

I think so. In my opinion documentations should aim to provide a pointer to behaviours of a system as far removed from the code as possible.

> Should we add a word on specific case ?

sure!

Reply:
cf https://github.com/ocaml/opam/pull/5636/files#r1389505791